omnisend/includes/Internal/class-sync.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Omnisend Contact Utils
+ *
+ * @package OmnisendClient
+ */
+
+namespace Omnisend\Internal;
+
+use Omnisend\SDK\V1\Contact;
+use Omnisend\SDK\V1\Omnisend;
+
+! defined( 'ABSPATH' ) && die( 'no direct access' );
+
+class Sync {
+
+	/**
+	 * @param int $limit number of users to sync
+	 */
+	public static function sync_contacts( int $limit = 100 ) {
+		$wp_user_query = new \WP_User_Query(
+			array(
+				'number'     => $limit,
+				'meta_query' => array( // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+					'relation' => 'OR',
+					array(
+						'key'     => UserMetaData::LAST_SYNC,
+						'compare' => 'NOT EXISTS',
+						'value'   => '',
+					),
+				),
+			)
+		);
+		$users         = $wp_user_query->get_results();
+
+		if ( empty( $users ) ) {
+			wp_clear_scheduled_hook( OMNISEND_CORE_CRON_SYNC_CONTACT );
```

Comment:
Would suggest to add log

Reply:
It is normal situation when all users are synced. We decided do not have UI and plugin does not have logs like Woo plugin. What log and where you expect?

Reply:
maybe adding logs now will save us time later, when we would introduce logs in UI.

Reply:
I was thinking about that, but decided not to do, as this will increase scope. Let's leave for later.